app/assets/stylesheets/_application.scss

```diff
@@ -1,7 +1,7 @@
 @import 'perfect-scrollbar';
 
 @import 'color-variables';
-@import "bootstrap-complete.scss";
+@import 'bootstrap-complete.scss';
```

Comment:
Imported partial `bootstrap-complete.scss` should be written as `bootstrap-complete`